delta/app/src/test/resources/organizations/org-resource.json

```diff
@@ -1,7 +1,7 @@
 {
   "@context": [
-    "https://bluebrain.github.io/nexus/contexts/resource.json",
-    "https://bluebrain.github.io/nexus/contexts/organizations.json"
+    "https://bluebrain.github.io/nexus/contexts/organizations.json",
+    "https://bluebrain.github.io/nexus/contexts/resource.json"
```

Comment:
we should standardize on plural

Reply:
Notice that resource is just the nexus metadata.  It is not a domain resource as organizations, projects, schemas,  etc. So maybe we can call it differently. Ideas?

Reply:
hmmm, good point; maybe renaming it to `metadata.json`?